### PR TITLE
WRR-1010: `Scroller`: Fixed to focus content area properly when `focusableScrollbar` prop is `byEnter`

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The last input type is not recognized properly so that Scroller does not focus the contents well when `focusableScrollbar` is `byEnter`.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
The issue occurs after #1576. When the new Enact API is added and Sandstone is changed to use it, the logic to check if the platform is supported became unclear. Now, we decided to let Enact to simply call webOS specific API without checking and to make Sandstone to check the availability before call the API since the feature it handles is just for specific platforms that Sandstone determines.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-1010
enactjs/enact#3274
Related: WRQ-6498 / #1576

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Fixed
 
 - `sandstone/Scroller`, `sandstone/Slider`, and `sandstone/VirtualList` to have default prop when `undefined` prop is passed
+- `sandstone/Scroller` to show the focus effect of the body on PC environments when `focusableScrollbar` prop is `byEnter`
```
Comment (MikyungKim) [inline CHANGELOG.md:19]:
```suggestion
- `sandstone/Scroller` to focus content area properly on supported platforms when `focusableScrollbar` prop is `byEnter`
```

File: ThemeDecorator/ThemeDecorator.js [hunk 2]
```diff
@@ -9,21 +9,23 @@ import {setDefaultTargetById} from '@enact/core/dispatcher';
 import {addAll} from '@enact/core/keymap';
 import hoc from '@enact/core/hoc';
```
Comment (MikyungKim) [inline ThemeDecorator/ThemeDecorator.js:10]: Please swap the rows.

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Fixed
 
 - `sandstone/Scroller`, `sandstone/Slider`, and `sandstone/VirtualList` to have default prop when `undefined` prop is passed
+- `sandstone/Scroller` to focus content area properly on supported platforms when `focusableScrollbar` prop is `byEnter`
```
Comment (MikyungKim) [inline CHANGELOG.md:19]: Would you please merge the latest release/2.9.x.develop into this branch?

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**